packages/ff/ff.0.2.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+authors: [ "Danny Willems <be.danny.willems@gmail.com>" ]
+maintainer: [ "Danny Willems <be.danny.willems@gmail.com>" ]
+synopsis: "OCaml implementation of Finite Field operations"
+description: "OCaml implementation of Finite Field operations"
+homepage: "https://gitlab.com/dannywillems/ocaml-ff"
+dev-repo: "git+https://gitlab.com/dannywillems/ocaml-ff.git"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-ff/issues"
+license: "MIT"
+depends: [
+  "dune" {build & >= "2.0"}
```

Comment:
Dune does not support the `{build}` tag. See https://github.com/ocaml/dune/issues/2147

Reply:
TBH, I am not sure I understand the consequence of this. Do you mean I should just remove the `build` tag?

Reply:
yes